src/main/java/rx/internal/operators/OperatorDistinct.java

```diff
@@ -29,6 +31,15 @@
  */
 public final class OperatorDistinct<T, U> implements Operator<T, T> {
     final Func1<? super T, ? extends U> keySelector;
+    
+    private static class Holder {
+        static final OperatorDistinct<?,?> INSTANCE = new OperatorDistinct<Object,Object>(UtilityFunctions.<Object>identity());
+    }
+    
+    @SuppressWarnings("unchecked")
```

Comment:
I'd add a javadoc comment here so it is more apparent the returned instance uses the identity function for comparison.